eZ/Bundle/EzPublishCoreBundle/Command/DebugConfigResolverCommand.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\VarDumper\Cloner\VarCloner;
+use Symfony\Component\VarDumper\Dumper\CliDumper;
+
+class DebugConfigResolverCommand extends ContainerAwareCommand
+{
+    /**
+     * {@inheritdoc}.
+     */
+    public function configure()
+    {
+        $this->setName('ezplatform:debug:config-resolver');
+        $this->setAliases(['ezplatform:debug:config']);
+        $this->setDescription('Debug / Retrive parameter from Config Resolver');
+        $this->addArgument(
+            'parameter',
+            InputArgument::REQUIRED,
+            'The configuration resolver parameter to return, for instance "languages" or "http_cache.purge_servers"'
+        );
+        $this->addOption(
+            'json',
+            null,
+            InputOption::VALUE_NONE,
+            'Only return value, for automation / testing use on a single line in json format.'
+        );
+        $this->addOption(
+            'scope',
+            null,
+            InputOption::VALUE_REQUIRED,
+            'Set another scope (siteaccess) to use, alternative to usiong the global --siteaccess[=SITEACCESS] option.'
+        );
+        $this->addOption(
+            'namespace',
+            null,
+            InputOption::VALUE_REQUIRED,
+            'Set another namespace then default "ezsettings" used by siteaccess settings.'
+        );
+        $this->setHelp(<<<EOM
+Outputs a given config resolver parameter, more commonly known as a SiteAccess setting.
+
+By default it will give value depending on global <comment>--siteaccess[=SITEACCESS]</comment> (default siteaccess is used if not set).
+
+However you can also manualy set <comment>--scope[=NAME]</comment> yourself if you don't want to affect the siteaccess
+set by the system. And you can also override namespace to get something else then default "ezsettings" namespace using
```

Comment:
>else then default

else th**a**n default.

It's not a strict rule, but maybe we can start a sentence without "And"?
> You can also override namespace...